rmw_connext_shared_cpp/src/qos.cpp

```diff
@@ -18,6 +18,21 @@
 namespace
 {
 
+bool
+is_time_default(const rmw_time_t & time)
+{
+  return time.sec != 0 && time.nsec != 0;
```

Comment:
(time.sec == 0 && time.nsec == 0) is default, right?

Reply:
you're right, i reversed the logic